### PR TITLE
Enhancement/2068 top content

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue #2068

## Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/googlesitekit/widgets/components/WidgetHeaderCTA.js [hunk 1]
```diff
@@ -38,12 +38,13 @@ const WidgetHeaderCTA = ( { href, label, external } ) => (
...
-	href: PropTypes.string.isRequired,
+	href: PropTypes.string,
 	label: PropTypes.string.isRequired,
 	external: PropTypes.bool,
 };
 
 WidgetHeaderCTA.defaultProps = {
+	href: '#',
 	external: true,
 };
```
Comment (felixarntz) [inline assets/js/googlesitekit/widgets/components/WidgetHeaderCTA.js:49]: This looks problematic, why did you add it? We should never use `#` for a `href` attribute; also why is it not required?

**Review by felixarntz — CHANGES_REQUESTED** [review]
Awesome work, this is almost ready. I just left one small comment, but there's one other disparity between the new and old widget we should look at: For whatever reason, the bounce rate I see differs _for some entries_.

They are already different in the response, so there must be some minor disparity between the new and old request made to Analytics. Potentially the date parsing logic on the server is slightly different? Although that would be odd because the other numbers all matched for me. Could you have a look?

**New widget**
Screen Shot 2021-03-25 at 10 12 25 AM

**Old widget**
Screen Shot 2021-03-25 at 10 12 38 AM

File: assets/js/googlesitekit/widgets/components/WidgetHeaderCTA.js (hunk 1 above)
Comment (eugene-manuilov) [reply assets/js/googlesitekit/widgets/components/WidgetHeaderCTA.js:49]: The `getServiceReportURL` selector returns undefined at the beginning. It causes console errors because `WidgetHeaderCTA` requires a valid URL. As a workaround, I have decided to drop `.isRequired` for the `href` attribute and set `#` by default.

Comment (felixarntz) [reply assets/js/googlesitekit/widgets/components/WidgetHeaderCTA.js:49]: @eugene-manuilov Can you use an empty string as default instead? That's what `SourceLink` does too.

Comment (eugene-manuilov) [reply assets/js/googlesitekit/widgets/components/WidgetHeaderCTA.js:49]: Ok, I'll update it.

Comment (eugene-manuilov) [reply assets/js/googlesitekit/widgets/components/WidgetHeaderCTA.js]: Updated.

**Comment by eugene-manuilov** [comment]
@felixarntz all should be good now.

**Review by felixarntz — APPROVED** [review]
Awesome!
